super_csv/mixins.py

```diff
@@ -95,7 +95,10 @@ def save(self, op_name=None):
         Save the state of this object to django storage.
         """
         state = self.__dict__.copy()
-        for k, v in state.items():
+        user = state.get('_user')
+
+        for k in list(state):
```

Comment:
Was this to fix a quality issue?

Reply:
this was to fix a python 3 issue. in python 3 you can't delete from a dictionary you're iterating through with .items()